src/Raven.Client/Documents/Smuggler/SmugglerResult.cs

```diff
@@ -338,6 +343,12 @@ public override string ToString()
                 if (ClientConfigurationUpdated)
                     sb.AppendLine("- Client");
 
+                if (TimeSeriesConfigurationUpdated)
+                    sb.AppendLine("- Time Series");
+
+                if (DocumentCompressionConfigurationUpdated)
+                    sb.AppendLine("- Document Compression");
```

Comment:
Documents Compression

Reply:
in studio we have Document Compression I matched it

Reply:
![image](https://user-images.githubusercontent.com/1250431/83147645-4351f680-a0f8-11ea-8060-da2e1c25fd13.png)

Reply:
done